Libplanet/KeyStore/IKeyStore.cs

```diff
@@ -0,0 +1,54 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+
+namespace Libplanet.KeyStore
+{
+    /// <summary>
+    /// The interface to store <see cref="ProtectedPrivateKey"/>s.  An appropriate implementation
+    /// should be used according to a running platform.
+    /// </summary>
+    public interface IKeyStore
+    {
+        /// <summary>
+        /// Lists all keys IDs in the key store.
+        /// </summary>
+        /// <returns>All keys IDs in the key store.  The order is undefined and not deterministic.
+        /// </returns>
+        public IEnumerable<Guid> ListIds();
+
+        /// <summary>
+        /// Lists all keys in the key store.
+        /// </summary>
+        /// <returns>All keys in the key store.  The order is undefined and not deterministic.
+        /// </returns>
+        public IEnumerable<Tuple<Guid, ProtectedPrivateKey>> List();
+
+        /// <summary>
+        /// Looks for a key having the requested <paramref name="id"/> in the key store.
+        /// </summary>
+        /// <param name="id">The key ID to look for.</param>
+        /// <returns>The found key.</returns>
+        /// <exception cref="NoKeyException">Thrown when there are no key of the given
+        /// <paramref name="id"/>.</exception>
+        public ProtectedPrivateKey Get(Guid id);
+
+        /// <summary>
+        /// Adds a new <paramref name="key"/> into the key store.
+        /// </summary>
+        /// <param name="key">A key to add.</param>
+        /// <returns>The id of the added <paramref name="key"/>.</returns>
+        /// <exception cref="ArgumentNullException">Thrown when <c>null</c> is passed to
+        /// <paramref name="key"/>.</exception>
+        public Guid Add(ProtectedPrivateKey key);
+
+        /// <summary>
+        /// Removes a key having the given <pramref name="id"/> from the key store.
+        /// </summary>
+        /// <param name="id">The key ID to remove.</param>
+        /// <exception cref="NoKeyException">Thrown when there is no key having
+        /// the given <paramref name="id"/>.</exception>
+        public void Remove(Guid id);
+    }
+}
+#nullable restore
```

Comment:
I think we can remove this.

Reply:
Is `#nullable` context per file?  Could you give any docs to address it?

Reply:
It is difficult to find, but it is written here.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-8.0/nullable-reference-types-specification#nullable-contexts

> A directive sets the context(s) it controls for subsequent lines of code, until another directive overrides it, or until the end of the source file.